src/lib/darkcoinClient.ts

```diff
@@ -154,6 +139,31 @@ export default class DarkcoinClient {
 
   // GObjects
 
+  /**
+   * The gobject prepare RPC prepares a governance object by signing and creating a collateral transaction.
+   * @param parentHash Hash of the parent object. usually the root node which has a hash of 0
+   * @param revision Object revision number (Always 0)
+   * @param creationTime Creation time as a unix timestamp
+   * @param gobjectData Object data (JSON object with governance details)
+   * @returns Transaction id for the collateral transaction
+   */
+  public gobjectPrepare(parentHash: string, revision: number, creationTime: number, gobjectData: string): Promise<CallResult<string>> {
+    return this.callRPCMethod<string>('gobject', ['prepare', parentHash.toString, revision.toString, creationTime.toString, gobjectData]);
+  }
+
+  /**
+   * The gobject submit RPC submits a governance object to network (objects must first be prepared via gobject prepare).
+   * @param parentHash Hash of the parent object. Usually the root node which has a hash of 0
+   * @param revision Object revision number (Always 0)
+   * @param creationTime Creation time as a unix timestamp (I think it needs to match the timestamp used in gobjectPrepare)
+   * @param gobjectData Object data (JSON object with governance details)
+   * @param txID Collateral transaction ID
+   * @returns The resulting governance object hash
+   */
+  public gobjectSubmit(parentHash: string, revision: number, creationTime: number, gobjectData: string, txId: string): Promise<CallResult<string>> {
+    return this.callRPCMethod<string>('gobject', ['submit', parentHash.toString, revision.toString, creationTime.toString, gobjectData, txId]);
```

Comment:
> parentHash.toString

It is redundant to use `toString` on a string

> revision.toString

I think this does not work. I believe this is going to pass the function itself, instead of a string. i.e. you need to call `revision.toString()` instead to apply the function.